oci/default.nix

```diff
@@ -101,7 +102,9 @@ entries // rec {
       # if we have an image after all this, hash it and build up the OCI blob
       # structure
       if [ -f $out/layer.tar ]; then
-        python ${./hash-layer.py} $out
+        sha256sum $out/layer.tar | cut -b -64 > $out/contenthash
+        pigz -3 -n -m $out/layer.tar
```

Comment:
We use `pigz` so that it compresses faster and `-3` seems to add minimal time and a slightly better compression compared to the automatic EKS compression. The `-n -m` flags prevent it from storing additional data to keep it mostly deterministic.